container/endpoint.go

```diff
@@ -211,7 +211,7 @@ func buildExportedEndpoints(conn coordclient.Connection, tenantID string, state
 			exp.endpointName = defep.Name
 
 			var err error
-			ep, err := buildApplicationEndpoint(state, &defep)
+			ep, err := dao.BuildApplicationEndpoint(state, &defep)
```

Comment:
I don't like the idea of having one-off methods in the DAO that aren't for the ControlPlaneDAO.  I think they should be in a different package altogether